programs/server/Server.cpp

```diff
@@ -993,23 +993,6 @@ try
     }
 #endif
 
-    /// When building openssl into clickhouse, clickhouse owns the configuration
```

Comment:
I have never seen XML tag "opensslconf" somewhere in the docs or tests, but perhaps you like to double-check and if yes, remove such references.

Reply:
I like that we are getting rid of the code below but I also asked myself if there is a point in checking env var `OPENSSL_CONF` in OpenSSLInitializer::initialize?

Reply:
It will make sense in case of FIPS build.